### PR TITLE
Changes in Brazilian Portuguese Translations

### PR DESCRIPTION
**Pull request by ejkun**
## Changelog

```markdown
### Fixed
- Fixed PT-BR translations

```

## Changes in Brazilian Portuguese Translations

Some translations were missing and others were incorrect

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
We need another Brazilian speaker here to validate this PR! :smile: 

Could you please fix your commit author?

**Comment by ejkun** [comment]
Fixed. If it is needed, I can explain all the translation changes.

**Comment by Soullivaneuh** [comment]
> If it is needed, I can explain all the translation changes.

Would be great, I'm not a Brazilian/Portuguese speaker. ;-)

File: Resources/translations/SonataAdminBundle.pt_BR.xliff [hunk 1]
```diff
@@ -164,11 +164,11 @@
             </trans-unit>
             <trans-unit id="flash_lock_error">
                 <source>flash_lock_error</source>
-                <target>Outro usuário modificou o item "%name%". Por favor, %link_start%clique aqui%link_end% to para recarregar a página e aplicar as mudanças novamente.</target>
+                <target>Outro usuário modificou o item "%name%". Por favor, %link_start%clique aqui%link_end% para recarregar a página e aplicar as mudanças novamente.</target>
             </trans-unit>
             <trans-unit id="flash_batch_delete_success">
                 <source>flash_batch_delete_success</source>
-                <target>Itens solucionados foram deletados com sucesso.</target>
+                <target>Itens selecionados foram deletados com sucesso.</target>
```
Comment (javiereguiluz) [inline Resources/translations/SonataAdminBundle.pt_BR.xliff]: This translation is specially important! Previously it said "solved items ..." and now it correctly says "selected items ..."

**Comment by javiereguiluz** [comment]
👍

**Comment by Soullivaneuh** [comment]
Thanks @ejkun for your contribution and @javiereguiluz for the review! :heart:

**Comment by ejkun** [comment]
- flash_lock_error
     I removed the "to" which means "para", because there is already a "para" in the translation. It was just a word someone forgot.

- flash_batch_delete_success
    the "solucionados" means "solved". The right is "selecionados" which means "selected".

- btn_preview
    The word "Previsualização" doesn't exist. The right is "Pré-visualização".

- label_unknown_user
    "Usuário" means "user" and "desconhecido" means "unknown". The difference in the words order is because in portuguese most adjectives come after the subject.

**Pull request merged**